test/protocols/test_unit_conversion_time_conflict.txt

```diff
@@ -0,0 +1,83 @@
+documentation {
+# Graph state variables over a single pace
+
+This tiny protocol simply simulates the model for 1 second from its initial conditions,
+and plots all the state variables over this time interval in a single graph.
+}
+
+
+# The model annotations used by this protocol to interface to models all live in the 'oxford-metadata' ontology
+# defined by the Web Lab team.  By convention we associate the full URI for this ontology with the 'oxmeta' prefix.
+# Terms from the ontology can then be accessed using the shorthand oxmeta:term syntax, as illustrated in the
+# documentation above.
+# The available terms can be seen in the model annotation tool in the Web Lab (available when viewing a model you
+# have uploaded yourself).
+
+namespace oxmeta = "https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#"
+
+# The standard library of post-processing operations is imported and associated with the 'std' prefix.
+
+import std = "BasicLibrary.txt"
+
+# Next we define the physical units used later.
+
+units {
+    mV = milli volt
+    ms = milli second
+}
+
+# The model interface section defines how the protocol can modify and probe the models to which it is applied.
+# Crucially, note that all references to model variables in this section use terms from the ontology declared above (with the
+# oxmeta prefix), rather than whatever variable names are used in the model.  Protocols can only access variables that are
+# annotated with terms from a declared ontology (typically the 'oxmeta' ontology for cardiac electrophysiology, although this
+# is not required by the tools).
+
+model interface {
+    # We specify the units this protocol uses for the independent variable in the system.
```

Comment:
So this is the only place where we need a comment, highlighting what is tested with these two lines.